style.css

```diff
@@ -479,8 +479,8 @@ button {
   display: 80px;
   flex-direction: column;
   margin-top: 100px;
-  background: url(img/contact-form-imga.png), #828adb;
-  background-size: 100% 135%;
+  background: url(img/contact-form-imga.png) #6070FF top right;
+  background-size: 50% 120%;
   background-repeat: no-repeat;
   flex-direction: column;
   align-items: center;
```

Comment:
## Well done implementing the changes requested by the previous reviewer 👏🏼 

Here are some points and improvements that can help your project to standout ✨ 

- **[Optional]** The display property has a specific values instead of `80px` please refer to [CSS display Property](https://www.w3schools.com/cssref/pr_class_display.php) to understand more about the topic 😉💪🏼 
- **[Optional]** The design looks good at some screen resolutions👌 Please be aware that the mobile screens could variates between **min 375px** to **max 768px** for that reason I highly recommend following the mobile design first approach 👍🏼 and You will find many valuable articles and lessons for this topic, however, the trick is **You just look at any screen even if it's large as its a mobile screen** 😎.
- **[Optional]** Please consider customizing your Readme file to be unique to this project 👍🏼 

### Contact section at **451px** 👍🏼 
![image](https://user-images.githubusercontent.com/68030297/222582605-435ac908-e5b7-4f95-8451-0f0d0ec4eb43.png)

### Contact section at **737px**
![image](https://user-images.githubusercontent.com/68030297/222583192-036d52dd-f80f-46be-b858-8a8fa47cea33.png)